test/passing/source.ml.ref

```diff
@@ -4814,9 +4800,7 @@ module A_alias_expanded = struct
   include A_alias
 end
 
-let _ = f (module A_alias_expanded : S with type t = (module A.A_S))
-
-(* ok *)
+let _ = f (module A_alias_expanded : S with type t = (module A.A_S)) (* ok *)
```

Comment:
Is this intended ? (Comment was on its line, separated by an empty line)

Reply:
In source.ml we have `let _ = f (module A_alias_expanded : S with type t = (module A.A_S)) (* ok *)` is also on the same line so this is a fix.